VadeSecure/vade_secure_m365/tests/email.json

```diff
@@ -0,0 +1,38 @@
+{
+    "input": {
+        "sekoiaio": {
+            "intake": {
+                "dialect": "vade-m365",
+                "dialect_uuid": "e4a758fc-7620-49e6-b8ed-b7fb3d7fa232"
+            }
+        },
+        "message": " {\"id\": \"zekfnzejnf576rge8768\", \"date\": \"2022-02-10T13:00:05.454Z\", \"sender_ip\": \"192.168.1.1\", \"from\": \"test@sekoia.io\", \"from_header\": \"<test@sekoia.io>\", \"to\": \"test@vadesecure.com\", \"to_header\": \"\\\"test@vadesecure.com\\\" <test@vadesecure.com>\", \"subject\": \"Lorem ipsum dolor\", \"message_id\": \"<01de2305-f75b-49db-8c61-f661bd498e63.protection.outlook.com>\", \"urls\": [{\"url\": \"https://sekoia.io\"}], \"attachments\": [], \"status\": \"LEGIT\", \"substatus\": \"\", \"remediation_type\": \"none\", \"remediation_ids\": [], \"action\": \"NOTHING\", \"folder\": \"\", \"size\": 113475, \"current_events\": [], \"whitelisted\": false} "
+    },
+    "expected": {
```

Comment:
```json
{
    "id": "zekfnzejnf576rge8768",
    "date": "2022-02-10T13:00:05.454Z",
    "sender_ip": "192.168.1.1",
    "from": "test@sekoia.io",
    "from_header": "<test@sekoia.io>",
    "to": "test@vadesecure.com",
    "to_header": "test@vadesecure.com <test@vadesecure.com>",
    "subject": "Lorem ipsum dolor",
    "message_id": "<01de2305-f75b-49db-8c61-f661bd498e63.protection.outlook.com>",
    "urls": [
      {
        "url": "https://sekoia.io"
      }
    ],
    "attachments": [],
    "status": "LEGIT",
    "substatus": "",
    "remediation_type": "none",
    "remediation_ids": [],
    "action": "NOTHING",
    "folder": "",
    "size": 113475,
    "current_events": [],
    "whitelisted": false
  }
  ```

I noted that some fields are not extracted (e.g. action, whitelisted,..) don't we need those fields ?

Reply:
`action` is used in `categorization` stage
For other, we can still add them if necessary when TDR will develop specific rules